### ISSUE TITLE
Accessibility issues with Table component

### ISSUE THREAD

**Issue by roelvangils** [digipolisantwerp/acpaas-ui_angular#231]
Erik and Roel reported back these WCAG issues.

### 1.1.1 Non-text Content

- Issue: the eyes that are used to indicate a closed or opened actions-menu do not have a state and fall for many of the [fontawesome](https://www.notion.so/Digipolis-5ed9b0aec4ed4f0e8d3684bdaa16c447#aea9265ed3b34ae3b24ec3e542235dc8) issues.

## Recommendations

The '#' as a column header is hard to understand for many people and does not get pronounced by screen readers. Writing out full words is preferred for clarity.

The action buttons all have the same `title` attribute. A proper image inside of the button with an accessible name or visible label makes the `title` attribute redundant. Give each button a unique accessible name if possible to differentiate them.

Consider using `<datetime>` for inputted dates.

The `table` does not have a `<caption>`. Adding this would greatly improve clarity.

The actions menu does not work in Firefox, Edge or Chrome on Windows 10.
